utils/__init__.py

```diff
@@ -47,3 +47,17 @@ def generate_invite_url(discord_bot_application_id: int, discord_guild_id: int)
         scopes=("bot", "applications.commands"),
         disable_guild_select=True,
     )
+
+def is_user_inducted(guild: discord.Guild, user: discord.Member) -> bool:
```

Comment:
The user parameter should have type `discord.Member | discord.User` for added flexibility. This will, however, require the additional retrieval of the actual member object if only a user object is given. Member retrieval can be done using the `TeXBot.get_main_guild_member` function.

Reply:
I think it'd be better to leave it as discord.Member - making it possible to pass in a user that isn't a member causes issues with how that error is then propagated back up so I'd suggest keeping it as member because you shouldn't be checking if a user you aren't sure is even in the server is inducted

Reply:
Very fair, I suppose if someone did need to perform a similar thing, they could use the `get_main_guild_member` function first then pass that member into this command.